### ISSUE TITLE
Format for Info.json

### ISSUE THREAD

**Issue by GeorgeBarlow** [jcts3/whenismytrain#5]
What is the format for the info.json?

**Comment by jcts3** [comment]
Hi George, sorry I didn't see this yesterday (still a bit of a noob with github). I'm at work atm so don't have the code, but aim to get back to you this evening.

**Comment by jcts3** [comment]
I've added a [info_template.json file](https://github.com/jcts3/whenismytrain/blob/master/info_template.json), so you can see the format it should be in - I stuck in a couple of dummy routes to the favourite services. 
I'll try and add a section to my How To section in the README.md file to explain briefly how to fill this out - though this might end up being over two commits as I'm a wee bit short on time at the moment! 
Finally, thanks for having a look at my project! I hope you can get it to work, and let me know if you have any other questions or suggestions 😄

**Comment by jcts3** [comment]
Instructions added [here](https://github.com/jcts3/whenismytrain/blob/master/README.md#instructions-for-file-creation)

**Comment by GeorgeBarlow** [comment]
Thanks a lot, looks great, trying to get it up and running now. Also it is just npm install then node app.js right, having a bit of trouble where node app.js doesn't do anything and just leaves an idented blank space in the command line :)

**Comment by jcts3** [comment]
That sounds about right, but maybe i should put in a log message at that point to say it's successfully started up...
So it's running an express app so you have to use your browser to view results. So for example going to http://localhost:8081/EDB/GLQ/departureinfo gives you the departure info for the next train from Edinburgh to Glasgow Queen Street

**Comment by jcts3** [comment]
(feel free to change the port value in app.js if you've got some service running on 8081)

**Comment by GeorgeBarlow** [comment]
Oh sorry my mistake, it's working fine :) Looks really cool 👍

**Comment by jcts3** [comment]
Cheers! Nowhere near finished compared to what I'd like to do but it's a start 😃 Maybe I should fix up some basic documentation before making things even more convoluted/complicated..

**Comment by GeorgeBarlow** [comment]
You could do something really cool with this 👍 and tbh only if you want people to contribute otherwise get the product done then do docs (thats usually how I role :) )

**Comment by jcts3** [comment]
Yeah, it got to a a point where I managed to set up a raspberry pi with it running in my house to check for when my train leaves in the morning, then went on holiday and have struggled to pick it back up haha. You've encouraged me enough to get back into it, so thanks for that 😀

**Comment by GeorgeBarlow** [comment]
Ah I’m glad :) it has very high potential, but if you ever get that raspberry pi thingy working again I’d be more than grateful for a setup guide 😁
